tools/check_ip_addresses.pl

```diff
@@ -43,7 +43,11 @@ END
   next if $ip =~ /192\.0\.2\./;         # TEST-NET-1 (RFC 5737)
   next if $ip =~ /198\.51\.100\./;      # TEST-NET-2 (RFC 5737)
```

Comment:
is there a reason to do this with regex and not to just:
```perl
  candidate = ipaddres.IPAddress($ip)
  result = false
  for net in netsDict {
      if candidate in net {
       result=true
       break
     }
  }

if !result errorBdaly
```

Reply:
@robshakir you might know?  (does not block this PR merge)

Reply:
I am not sure who wrote this perl originally -- honestly, if we refactor it, should we just make it not-perl? :-)